pontoon/base/tests/test_models.py

```diff
@@ -0,0 +1,75 @@
+from django_nose.tools import assert_equal
+
+from pontoon.base.models import Translation
+from pontoon.base.tests import (
+    assert_redirects,
+    EntityFactory,
+    LocaleFactory,
+    ProjectFactory,
+    ResourceFactory,
+    TranslationFactory,
+    UserFactory,
+    TestCase
+)
+from pontoon.base.utils import aware_datetime
+
+
+class LocaleTests(TestCase):
+    def test_latest_activity_translated(self):
+        """
+        If latest activity in Translation QuerySet is translation submission,
+        return submission date and user.
+        """
+        user0 = UserFactory.create()
+        user1 = UserFactory.create()
+
+        translation0 = TranslationFactory.create(
```

Comment:
Descriptive variable names make tests a lot easier to understand (I used to name test variables like this but I ended up getting confused a lot). In this case, `latest_submission` and `latest_approval` would be good names for these translations.